bench-vortex/src/tpch/duckdb.rs

```diff
@@ -94,15 +94,10 @@ pub fn generate_tpc(opts: DuckdbTpcOptions) -> Result<PathBuf> {
         return Ok(output_dir);
     }
 
-    let is_local_duckdb = opts.duckdb_path.is_some();
-
     let mut command = Command::new(opts.duckdb_path.unwrap_or_else(|| PathBuf::from("duckdb")));
-
-    let vortex_path = vortex_duckdb_extension_path();
     command
-        .arg("-unsigned")
         .arg("-c")
-        .arg(format!("load \"{}\";", vortex_path.to_string_lossy()));
+        .arg("install vortex from community; load vortex;");
```

Comment:
I am not sure we want to do this this will regress all our benchmarks

Reply:
You can see this in clickbench

Reply:
Not about scanning.